vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerViewModel.kt

```diff
@@ -188,6 +189,9 @@ class MessageComposerViewModel @AssistedInject constructor(
 
     private fun handleSendMessage(action: MessageComposerAction.SendMessage) {
         withState { state ->
+            analyticsTracker.capture(state.toAnalyticsComposer()).also {
+                setState { copy(startsThread = false) }
```

Comment:
I'm wondering if there's a way to avoid this mutable state but I guess we need to be sure we only track the first message